components/script/dom/readablestreamdefaultreader.rs

```diff
@@ -41,28 +45,81 @@ impl ReadableStreamDefaultReader {
         proto: Option<SafeHandleObject>,
         stream: DomRoot<ReadableStream>,
     ) -> Fallible<DomRoot<Self>> {
-        todo!()
```

Comment:
Can we add a doc comment to this method that link the spec of this constructor?

Reply:
added